stockfish/models.py

```diff
@@ -200,6 +207,7 @@ def set_elo_rating(self, elo_rating: int = 1350) -> None:
             None
         """
         self._set_option("UCI_LimitStrength", "true")
+        self._parameters.update({"UCI_LimitStrength": "true"})
```

Comment:
These updates to parameters are needed, because if you set an option, but then call get_parameters() it will list the old parameter. This was causing some unit tests to fail.